docs/source/sysadmin_guide/configure_sgx.rst

```diff
@@ -402,6 +403,13 @@ specify the correct bind interface.
         Jun 02 14:50:37 ubuntu systemd[1]: sawtooth-validator.service: Unit entered failed state.
         Jun 02 14:50:37 ubuntu systemd[1]: sawtooth-validator.service: Failed with result 'exit-code'.
 
+Restrict permssions on validator.toml to protect the network private key.
```

Comment:
Please put double-backticks around validator.toml to make it a fixed-width font. (This is used for files, paths, code items, etc.) For example:
     \`\`validator.toml\`\`   

Email note: The \ characters are to protect the backticks in this GitHub comment.  Don't use \ with ` in the RST file.

Reply:
@chenette PR updated and rebased. Good catch, thanks!